hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -147,38 +151,30 @@ spec = parallel $
                 }
           }
 
-defaultOptions :: Options
-defaultOptions =
-  Options
-    { verbosity = Verbose "HydraNode"
-    , nodeId = 1
-    , host = "127.0.0.1"
-    , port = 5001
-    , peers = []
-    , apiHost = "127.0.0.1"
-    , apiPort = 4001
-    , monitoringPort = Nothing
-    , hydraSigningKey = "hydra.sk"
-    , hydraVerificationKeys = []
-    , chainConfig = defaultChainConfig
-    , ledgerConfig = defaultLedgerConfig
-    }
-
-defaultChainConfig :: ChainConfig
-defaultChainConfig =
-  DirectChainConfig
-    { networkId = Testnet (NetworkMagic 42)
-    , nodeSocket = "node.socket"
-    , cardanoSigningKey = "cardano.sk"
-    , cardanoVerificationKeys = []
-    }
-
-defaultLedgerConfig :: LedgerConfig
-defaultLedgerConfig =
-  CardanoLedgerConfig
-    { cardanoLedgerGenesisFile = "genesis-shelley.json"
-    , cardanoLedgerProtocolParametersFile = "protocol-parameters.json"
-    }
+    it "parses --start-chain-from as a pair of slot number and block header hash" $
+      ["--start-chain-from", "1000.0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"]
+        `shouldParse` defaultOptions
+          { chainConfig =
+              defaultChainConfig
+                { startChainFrom =
+                    Just
+                      ( ChainPoint
+                          1000
+                          (unsafeDeserialiseFromRawBytesBase16 "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef")
+                      )
+                }
+          }
+
+    prop "roundtrip options" $
+      forAll arbitrary canRoundtripOptionsAndPrettyPrinting
+
+canRoundtripOptionsAndPrettyPrinting :: Options -> Property
+canRoundtripOptionsAndPrettyPrinting opts =
```

Comment:
Pretty printing? Name should rather include `toArgs` .. if this name refers even to the use of that function (I'm not sure).